lms/static/sass/views/_program-details.scss

```diff
@@ -443,6 +443,34 @@ $btn-color-primary: $primary-dark;
     }
   }
 
+  .upgrade-subscription {
+    margin: 10px 15px 10px 5px;
+    gap: 15px;
+  }
+
+  .subscription-icon-restart {
+    width: 24px;
+    height: 24px;
+    margin-inline-end: 8px;
+  }
+
+  .subscription-icon-arrow-upright {
+    display: flex;
+    align-items: center;
+    width: 15px;
+    height: 15px;
+    margin-inline-start: 8px;
+  }
+
+  .subscription-info {
+    font-size: 0.9375em;
+    color: $gray-500;
+  }
+
+  .subscription-info-upsell {
+    margin-top: 0.25rem;
+    font-size: 0.8125em;
+  }
```

Comment:
for the font size, existing code was primarily using `em`, so I just followed that,
for the margin, initially i used the bootstrap `mt-1` class, but since i created a css class for font size, i just replaced the `mt-1` with equivalent style `margin-top: 0.25rem`